python_on_whales/components/network/cli_wrapper.py

```diff
@@ -209,6 +210,26 @@ def disconnect(
         full_cmd += [network, container]
         run(full_cmd)
 
+    def exists(
+        self,
+        network: str,
```

Comment:
Should support passing a `Network` object too, i.e. this should be `ValidNetwork` as with other methods such as `disconnect()` above.

Reply:
Updated, along with making sure the test covers that use case.